ratis-test/src/test/java/org/apache/ratis/datastream/TestDataStream.java

```diff
@@ -140,7 +189,7 @@ public void runTestDataStream(){
     final int bufferSize = 1024*1024;
     final int bufferNum = 10;
     final DataStreamOutput out = client.stream();
-    DataStreamClientImpl.DataStreamOutputImpl impl = (DataStreamClientImpl.DataStreamOutputImpl) out;
+    impl = (DataStreamClientImpl.DataStreamOutputImpl) out;
```

Comment:
Please revert the unnecessary change to make the code easy to review. If you want to refactor it, you can create another PR.

Reply:
oops. Done!